README.md

```diff
@@ -106,7 +106,7 @@ optional arguments:
                         will then fall back to looking for standard files in
                         the current directory that relate to the type of input
                         indicated by the -t flag.
-  -t TYPE, --type TYPE, -it TYPE, --input-type TYPE
+  -it TYPE, --type TYPE, --input-type TYPE
```

Comment:
After removing the `-t` option, do the examples starting on line 127, 134, 321, 331,  in this file still work? They appear to use the `-t`, so maybe need to be updated (or restore `-t` option)? 

If not restoring `-t`, I think the "usage" doc (line 96) also needs to be copy/pasted (updated) with latest args.

(Also just as likely I'm missing something here). ;)

Reply:
Hi, thanks for the review!
You're right, I didn't really check in-depth  the usage of that option in the documentation :sweat_smile: 
 
I made this change to be able to use the same switch on all the commands, but maybe it's better to only change `iq`s `--stage` option - what do you think about this?

Reply:
I'm leaning towards only changing the IQ `--stage` option, if only because it feels less disruptive.

Reply:
done, `iq` stage option renamed to  `-st`. I also forgot to update the `-i` -> `a` switch in the docs :sweat_smile:  fixed